scripts/e2e.windows.sh

```diff
@@ -14,12 +14,12 @@ cd windows_test
 # Install web3 as dep
 npm init --yes
 npm install web3@e2e --save --registry http://localhost:4873
-node ./basic_usage.js
+./basic_usage.js
 
 # Shutdown verdaccio server
 cd ..
 source verdaccio_pid
 kill -9 $VERDACCIO_PID
 
-# Debugging...
-ps -ef
+# Terminate stray node process
+kill -9 $(grep node <(ps -ef) | sed 's/.*\travis *\([0-9]*\).*/\1/')
```

Comment:
Hmm, these kind of things seem pretty dangerous to me if run in a local environment. E.g. I tested on my machine and the non-`kill` part returned a totally unrelated "Atom Helper" process having something with `node` in the passed CLI parameters.

Is there any way to make this more robust to avoid side effects like this?

Reply:
@holgerd77 

Oh yes...do you think checking that the `CI` environment variable is set before running this would be enough? 

Here's [a list of env variables set in TravisCI][1]

[1]: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Reply:
Added a check for CI in 2f1ebc5bd8a36bdfab651a09cd6be4086c4837ca